server.js

```diff
@@ -212,28 +194,56 @@ function searchMovieData(request, response){
 					let overview = movieObj.overview;
 					let average_votes = movieObj.vote_average;
 					let total_votes = movieObj.vote_count;
-					let image_url = movieObj.poster_path;
+					let img_url = movieObj.poster_path;
 					let popularity = movieObj.popularity;
 					let released_on = movieObj.release_date;
-					let location_id = request.query.data.id;
-					
-					const responseMovieObject = new MovieData(title, overview, average_votes, total_votes, image_url, popularity, released_on);
-		
-					client.query(SQL_CMDS.insertMovies, [responseMovieObject.title, responseMovieObject.overview, responseMovieObject.average_votes, responseMovieObject.total_votes, responseMovieObject.image_url, responseMovieObject.popularity, responseMovieObject.released_on, location_id])
-		
+
+					const responseMovieObject = new MovieData(title, overview, average_votes, total_votes, img_url, popularity, released_on, location_id);

```

Comment:
The work you are doing with lines 193 to 199 could be moved to the MovieData constructor function if it was refactored appropriately. Encapsulating the separation of these properties inside the constructor function could keep the flow of this section a little easier to follow.